docs/admin/kubernetes-ops/external-secrets-configmaps.md

```diff
@@ -121,6 +122,71 @@ kubectl get configmap -n <namespace> cn -o json
 
 From the console, Go to `Secret Manager`> Click on `Create Secret` > Add a `name` > Upload a `json` file or add the json to the `Secret value` field > Create
 
+### Vault
```

Comment:
Indicate the mode used for Vault installation. This is to clearly state what type of Vault installation was used because the cloud is slightly different